src/eko/kernels/non_singlet_qed.py

```diff
@@ -30,12 +30,12 @@ def as1aem1_exact(gamma_ns, a1, a0, aem, nf):
     """
     return np.exp(
         (gamma_ns[1, 0] + aem * gamma_ns[1, 1]) * ei.j12(a1, a0, aem, nf)
-        + aem * gamma_ns[0, 1] * ei.j02(a1, a0, aem, nf)
+        + aem * gamma_ns[0, 1] * np.log(mu2_from / mu2_to)
```

Comment:
Should we drop in consequence the `j0x` from the code?

Reply:
I think yes